calcom/cal.com
# fix: 404 links issue

## What does this PR do?

Links in 404 were not working, this fixes that

Fixes #9925 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Try going to a non-existent user such as `/asdffdsa` and see if links to register it and popular pages are working or not. Also see the community link and icon refer to Discord in 404 and in the app under the user dropdown.
